e2e/e2etests/e2etests.go

```diff
@@ -245,6 +247,12 @@ var AllE2ETests = []runner.E2ETest{
 		},
 		TestETHDepositAndCall,
 	),
+	runner.NewE2ETest(
+		TestETHDepositFastConfirmationName,
+		"deposit Ether into ZEVM using fast confirmation",
+		[]runner.ArgDefinition{},
+		TestETHDepositFastConfirmation,
+	),
```

Comment:
You probably need a MinimumVersion guard on this

Reply:
sound good. Added https://github.com/zeta-chain/node/pull/3551/commits/6a063b9e5a17631ef2989d3b7eb38cb3781ddac9